README.md

```diff
@@ -57,9 +58,19 @@ If you'd like to develop on and build the riddc book, you should:
 - Start a python virtual environment: 
   Mac/Unix: `python3 -m venv ./` 
   Windows: `python3 -m venv .\`
+- Activate the environment with `source ./bin/activate`(Mac/Linux) or 
+`source .\bin\activate` (Windows)
+
+## Install system dependencies
+- On Mac, download Homebrew and install with `brew bundle`
+- On Windows, download Chocolatey and install with `choco install packages.config`
```

Comment:
Have you tested the windows install?

Reply:
I added some instructions for Windows users to install HDF5 and added netcdf4 to `requirements.txt` so it will be install by `pip`. That works on Windows.